src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -95,6 +97,35 @@ private ApiMethodDocView generateOptionalArrayMethodDoc(MethodTransformerContext
     return docBuilder.build();
   }
 
+  /*
+   * Constructs a list of the optional fields for the method. Adds the page token field, which is
+   * missing from the optional fields list, and rearrange so that page token and page size are last.
```

Comment:
Uploaded a fix. I removed the reordering of optional args as well. Not sure if we want to leave that in?